client/src/components/ActionBar.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from "classnames";
+import Button from 'components/Button/Button';
+
+/**
+ * It is used to ship action buttons
+ *
+ * @param actions A array of actions that will be rendered as buttons
+ * @param isBottomActionBar Whether this bar should be placed at the bottom =
+ * @returns {JSX.Element}
+ * @constructor
+ */
+const ActionBar = ({actions, isBottomActionBar}) => {
+  const actionBarClass = classNames({
+    "action-bar": true,
+    "fill-width": true,
+    "action-bar--bottom": isBottomActionBar
+  });
+
+  const actionButtons = actions.map((action) => (
+      <Button {...action}>
```

Comment:
I think this thing will throw a warning if you don't give a unique `key` prop to each button.

Reply:
Ah, yeah, I should make the `key` prop to be required. Thanks!